### ISSUE TITLE
Datasets/tasks to show gnn usefulness vs other methods

### ISSUE THREAD

**Issue by nishanthcgit** [graph4ai/graph4nlp#405]
## ❓ Questions and Help

Hi all, I've been experimenting with graph4nlp and so far its been great. I've been learning a lot and really appreciate all the features built so far. As a new entrant to graph neural nets its been really easy to pick up.
While taking a look at the benchmarks though I had a question - when we compare bleu scores/whatever metric got using GNNs vs SotA approaches like transformers the latter seem to perform consistently better, often by significant margins. I wanted to know if there is an example of an NLP task where GNNs are able to use the structure of the graph to beat transformers based on the task parameters. 
Basically, looking for some motivation to use graphs for NLP vs current models like transformers.

**Comment by xiao03** [comment]
Hey @Nicoleqwerty, thanks for the questions. We are now on this issue. Please stay tuned for a response!

**Comment by hugochan** [comment]
> ## ❓ Questions and Help
> Hi all, I've been experimenting with graph4nlp and so far its been great. I've been learning a lot and really appreciate all the features built so far. As a new entrant to graph neural nets its been really easy to pick up.
> While taking a look at the benchmarks though I had a question - when we compare bleu scores/whatever metric got using GNNs vs SotA approaches like transformers the latter seem to perform consistently better, often by significant margins. I wanted to know if there is an example of an NLP task where GNNs are able to use the structure of the graph to beat transformers based on the task parameters.
> Basically, looking for some motivation to use graphs for NLP vs current models like transformers.

@nishanthcgit thanks for your interest to our library! Personally, I did observe GNN-based models outperformed transformer-based models in many NLP applications [[1](https://openreview.net/pdf?id=zv-typ1gPxA), [2](https://arxiv.org/pdf/2004.06015.pdf)] because the former could better utilize the rich structure of the input data.

Can you please share more details about your findings? For example, on which task and dataset you observed transformer-based models significantly outperformed the GNN-based models? What are the specific model architectures? How did you construct the graph for GNN-based models? These can help us better support you in applying GNNs to your task.

**Comment by xiao03** [comment]
Issue closed due to no response from the user.
